src/components/aboutModal/__tests__/__snapshots__/aboutModal.test.js.snap

```diff
@@ -2,50 +2,58 @@
 
 exports[`AboutModal Component should contain brand: brand 1`] = `
 <AboutModal
-  altLogo="Ipsum"
-  className=""
-  logo="logo-brand.svg"
-  onHide={[Function]}
-  productTitle={
+  appendTo={null}
+  backgroundImageSrc=""
+  brandImageAlt="Ipsum"
+  brandImageSrc="logo-brand.svg"
+  className="quipucords-about-modal"
+  isOpen={true}
+  noAboutModalBoxContentContainer={false}
+  onClose={[Function]}
+  productName={
     <img
       alt="Lorem ipsum"
       src="title-brand.svg"
     />
   }
-  show={true}
-  trademarkText="Copyright (c) 2019 Red Hat Inc."
+  trademark="t(about-modal.copyright, {\\"year\\":\\"2022\\"})"
```

Comment:
required:
confirm this isn't pulling a the actual year since this would fail the snapshot in the future

Reply:
Can confirm the testing date is set for `20220601`